src/test/kotlin/uk/gov/justice/digital/hmpps/hmppsinterventionsservice/service/ReferralServiceUnitTest.kt

```diff
@@ -209,6 +209,7 @@ class ReferralServiceUnitTest {
       )
 
       whenever(referralDetailsRepository.findLatestByReferralId(referral.id)).thenReturn(existingDetails)
+      whenever(authUserRepository.save(referral.createdBy)).thenReturn(referral.createdBy)
```

Comment:
_suggestion, not a blocker_
We can also use the pass-through-first-argument pattern, so tests aren't that coupled to the implementation:
```kt
whenever(authUserRepository.save(any())).then(AdditionalAnswers.returnsFirstArg<AuthUser>())
```